packages/eslint-plugin/package.json

```diff
@@ -12,9 +12,11 @@
     "@babel/core": "^7.12.13",
     "@typescript-eslint/utils": "^5.47.0",
     "csstype": "^3.1.2",
+    "synckit": "^0.8.5",
     "tslib": "^2.1.0"
   },
   "peerDependencies": {
-    "eslint": "^7.5.0 || ^8.0.0"
+    "eslint": "^7.5.0 || ^8.0.0",
+    "stylelint": "15.x"
```

Comment:
This probably should be a direct dependency, actually.

Reply:
well it's probably never going to be put into a bundle, so I don't mind